cli/cmd/middleware.go

```diff
@@ -96,3 +96,12 @@ func (p *resourceParameters) Validate(cmd *cobra.Command, args []string) []error
 
 	return nil
 }
+
+type paramError struct {
```

Comment:
moving this struct here to avoid having too tiny files poluting the dir view

Reply:
also unexporting, because this only makes sense in this context and has no external consumer (nor should it)